### PR TITLE
Updating Install instructions

### PR DESCRIPTION
**Pull request by aphelionz**
This PR adds dependencies to the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,26 @@ For more info see: https://docs.ipfs.io/introduction/overview/
...
+Rust IPFS depends on `protoc` and `openssl`.
+
+### Dependencies
+
+First, install the dependencies.
+
+With apt:
+
+```bash
+# apt-get install protobuf-compiler libssl-dev
```
Comment (koivunej) [inline README.md:56]: I assume this is hash to symbolize the root prompt? I haven't used root shell for so long time :)

Comment (aphelionz) [reply README.md:56]: You assume correctly :D

**Comment by koivunej** [comment]
Just in case someone comes wondering here later on: prost is used by rust-libp2p and rust-ipld and ... we may be using it as well.

**Comment by koivunej** [comment]
bors retry

**Comment by koivunej** [comment]
Failure cause was #194

**Comment by koivunej** [comment]
bors retry

**Comment by koivunej** [comment]
Lets get #212 in first, then start another round of retries.

**Pull request merged**